src/crimson/os/seastore/seastore_types.h

```diff
@@ -785,6 +785,7 @@ std::ostream &operator<<(std::ostream &out, extent_types_t t);
 struct extent_t {
   extent_types_t type;  ///< type of extent
   laddr_t addr;         ///< laddr of extent (L_ADDR_NULL for non-logical)
+  ceph::real_clock::time_point last_modified; ///< time of the last modification
```

Comment:
I'm wondering if it is possible&better to consolidate `last_modified` into the record header, for two reasons:
* Extents within the same user transaction have the same modification time;
* We are inclined to clean and move the extents of the similar coldness into the same segment, which means that the extents within the same cleaner transaction are preferred to have similar modification times.

Reply:
I think so, will try:-)